kola/harness.go

```diff
@@ -151,8 +164,20 @@ func runTest(t *Test, pltfrm string) error {
 	for k := range t.NativeFuncs {
 		names = append(names, k)
 	}
+
+	// prevent unsafe access if tests ever become parallel and access
+	tempTestOptions := make(map[string]string, 0)
```

Comment:
i'm not sure that this will ever be a problem in practice, because the options are written when the program starts, and (hopefully!) never written again. concurrent readers should be OK without locking.